### DISCUSSION TITLE
Token has invalid claims: token is expired

### DISCUSSION THREAD

**Discussion by maheshkumar2150** [golang-jwt/jwt#372] [Q&A] [answered]
How do i exactly look for the error?

What does it mean that token has invalid claims?

If the token has just expired, i would like to issue a new token. But it has invalid key or anything else, i cant issue a new token.

How do i check for errors?
Please give me an example

Thanks..

**Comment by oxisto** [comment, marked as answer]
I am not 100 % sure, if I understand your problem correctly, but if you are looking for a way to differentiate between different error messages coming from the parser, you can have a look at our examples:

https://github.com/golang-jwt/jwt/blob/4d0edcd99cd92b42b1a87ffe2063103d41ef3558/example_test.go#L164-L189

Basically, you can use `errors.Is` to check for the different error types.

**Reply by maheshkumar2150** [reply]
This is what i want exactly..

From where can i find this example?? 

Thanks a lot for helping..

**Reply by oxisto** [reply]
https://github.com/golang-jwt/jwt/blob/main/example_test.go
